src/index.ts

```diff
@@ -193,37 +194,36 @@ export class Translate extends common.Service {
     input = arrify(input);
 
     this.request(
-      {
-        method: 'POST',
-        uri: '/detect',
-        json: {
-          q: input,
+        {
```

Comment:
This went to a 4-space indent. Did a tool do that automatically?

Reply:
Yes, this was `clang-format`

Reply:
Follow-up! Is that what we want / is it configurable / who cares?

Reply:
For now, yeah it's want we want.  In fullness of time, we want to switch to prettier.  Discussion here:
https://github.com/GoogleCloudPlatform/google-cloud-node/issues/2842